monitoring/rid_qualifier/utils.py

```diff
@@ -0,0 +1,95 @@
+    ''' This object holds basic information about a point on the flight track, it has latitude, longitude and altitude in WGS 1984 datum '''
+
+    lat: float # Degrees of latitude north of the equator, with reference to the WGS84 ellipsoid. For more information see: https://github.com/uastech/standards/blob/master/remoteid/canonical.yaml#L1160
+    lng: float # Degrees of longitude east of the Prime Meridian, with reference to the WGS84 ellipsoid. For more information see: https://github.com/uastech/standards/blob/master/remoteid/canonical.yaml#L1170
+    alt: float # meters in WGS 84, normally calculated as height of ground level in WGS84 and altitude above ground level
+    
+
+class GridCellFlight(NamedTuple):
+    ''' A object to hold details of a grid location and the track within it '''
+    bounds: shapely.geometry.polygon.Polygon
+    track: List[FlightPoint]
+
+class RIDSP(NamedTuple):
+
+    ''' This is the object that stores details of a USS, mainly it will hold the injection endpoint and details of the flights allocated to the USS and their submissiion status '''
+
+    test_id: str
+    name: str
+    flight_id: int
+    rid_state_injection_url: str
+    rid_state_submission_status: bool
+
+
+class OperatorLocation(NamedTuple):
+    ''' A object to hold location of the operator when submitting flight data to USS '''
+    lat: float
+    lng: float
+
+class RIDFlightDetails(NamedTuple):
+    ''' A object to hold RID details of a flight operator that will be reported by the USS as a part of the test ''' 
+    operator_id:str
+    operation_description: str
+    operator_location: OperatorLocation
+    serial_number: str
+    registration_number: str
+
+class TestFlightDetails(NamedTuple):
+    ''' A object to hold the remote ID Details,  and a date time after which the USS should submit the flight details, it matches the TestFlightDetails in the injection interface, for more details see: https://github.com/interuss/dss/blob/master/interfaces/automated-testing/rid/injection.yaml#L158 ''' 
+    effective_after: datetime
+    details: RIDFlightDetails
+
+
+class TestFlight(NamedTuple):
+    ''' Represents the data necessary to inject a single, complete test flight into a Remote ID Service Provider under test; matches TestFlight in injection interface ''' 
+
+    injection_id: str    
+    telemetry: List[RIDAircraftState]
+    details_responses : TestFlightDetails   
+
+
+class TestPayload(ImplicitDict):
+    ''' This object defines the detail of a test object, one or more flight tracks may be assigned in a test therefore the requested flights is a list. '''
+
+    test_id: str 
+    requested_flights:List[TestFlight]
+
+class DeliverablePayload(ImplicitDict):
+    ''' This object defines the payload that needs will be submitted to the Test Inejection URL. The payload is a set of flight tracks, operator details and other associated objects. '''
+        
+    injection_path: str    
+    injection_payload: List[TestPayload]
```

Comment:
Currently, this field is describing multiple payloads yet it is named as if it contains a single payload.  We should pick a single interpretation and make sure all the names match.  In the case that `injection_payloads` is plural, then `DeliverablePayload` should be plural also as it contains many payloads.

Reply:
OK I have made the modifications but plural `DeliverablePayloads` as a ImplicitDict in the utils file sounds a bit off